{
  "comments": [
    {
      "key": {
        "uuid": "04cfc775_58646528",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 17,
      "author": {
        "id": 585
      },
      "writtenOn": "2020-10-05T15:15:59Z",
      "side": 1,
      "message": "this command also needs to filter the .tox directory, otherwise I get tons of errors:\n\n  -   bash -c \"yamllint {toxinidir}\"\n  +   bash -c \"find {toxinidir} \\\n  +   -not -path {toxinidir}/.tox/\\* \\\n  +   -name \\*.yaml | xargs yamllint\"",
      "range": {
        "startLine": 17,
        "startChar": 0,
        "endLine": 17,
        "endChar": 33
      },
      "revId": "0bfe948f2b8879ea4efb550aa5c36d93bd5f5492",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "cb89100b_f254ee4b",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 17,
      "author": {
        "id": 7019
      },
      "writtenOn": "2020-10-05T20:53:39Z",
      "side": 1,
      "message": "I applied this change on the yamllint config file(https://gerrit.opnfv.org/gerrit/c/kuberef/+/71190/2..3/.yamllint)",
      "parentUuid": "04cfc775_58646528",
      "range": {
        "startLine": 17,
        "startChar": 0,
        "endLine": 17,
        "endChar": 33
      },
      "revId": "0bfe948f2b8879ea4efb550aa5c36d93bd5f5492",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "6831f8e5_37b456d2",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 23,
      "author": {
        "id": 585
      },
      "writtenOn": "2020-10-05T15:15:59Z",
      "side": 1,
      "message": "I seem to need a different method to fix execution in a python 3.x environment. It still defaults to python 2.7 on my system, causing this test to fail.",
      "revId": "0bfe948f2b8879ea4efb550aa5c36d93bd5f5492",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "617187d7_5c3ddf43",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 23,
      "author": {
        "id": 7019
      },
      "writtenOn": "2020-10-05T20:53:39Z",
      "side": 1,
      "message": "What about using basepython, does that fix the problem?",
      "parentUuid": "6831f8e5_37b456d2",
      "revId": "0bfe948f2b8879ea4efb550aa5c36d93bd5f5492",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "6c6fe368_63636019",
        "filename": "tox.ini",
        "patchSetId": 2
      },
      "lineNbr": 23,
      "author": {
        "id": 585
      },
      "writtenOn": "2020-10-07T20:31:24Z",
      "side": 1,
      "message": "The Ubuntu 16.04 repos natively support up to python 3.5 which is not sufficient. If I manually install python3.6 (which is fine) and specify basepython \u003d python3.6, then everything works for me. It seems that basepython does not allow to specify a minimum version, such as \u003e\u003dpython3.6. So, do we want to fix this to one specific version of Python 3?",
      "parentUuid": "617187d7_5c3ddf43",
      "revId": "0bfe948f2b8879ea4efb550aa5c36d93bd5f5492",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    }
  ]
}